doc/src/build/rust-sdk.md

```diff
@@ -3,25 +3,43 @@ title: Sui Rust SDK
 ---
 
 ## Overview
-The Sui SDK is a collection of rust JSON-RPC wrapper and crypto utilities that you can use to interact with the Sui Gateway and Sui Full Node.
-The `SuiClient` can be used to create a http(`SuiClient::new_http_client`) or a websocket client(`SuiClient::new_ws_client`).  
-See [JSON-RPC doc](json-rpc.md#sui-json-rpc-methods) for list of available methods.
+The [Sui SDK](https://github.com/MystenLabs/sui/tree/main/crates/sui-sdk) is a collection of Rust language JSON-RPC wrapper and crypto utilities you can use to interact with the [Sui Devnet Gateway](../explore/devnet.md) and [Sui Full Node](fullnode.md).
 
-> Note: As of v0.6.0, the web socket client is for subscription only, please use http client for other api methods.
+The [`SuiClient`](cli-client.md) can be used to create an HTTP (`SuiClient::new_http_client`) or a WebSocket client(`SuiClient::new_ws_client`).  
+See our [JSON-RPC](json-rpc.md#sui-json-rpc-methods) doc for the list of available methods.
 
-## Examples
-Add the sui-sdk crate in your Cargo.toml:
+> Note: As of [Sui version 0.6.0](https://github.com/MystenLabs/sui/releases/tag/devnet-0.6.0), the WebSocket client is for [subscription only](pubsub.md); use the HTTP client for other API methods.
+
+## References
+
+Find the `rustdoc` output for key Sui projects at:
+
+* Sui blockchain - https://mystenlabs.github.io/sui/
+* Narwhal and Tusk consensus engine - https://mystenlabs.github.io/narwhal/
+* Mysten Labs infrastructure - https://mystenlabs.github.io/mysten-infra/
+
+## Configuration
+Add the `sui-sdk` crate in your `Cargo.toml` like so:
 ```toml
 [dependencies]
 sui-sdk = { git = "https://github.com/MystenLabs/sui" }
 ```
-Use the devnet branch if you are connecting to the devnet. 
+If you are connecting to the devnet, use the `devnet` branch instead:
 ```toml
 [dependencies]
 sui-sdk = { git = "https://github.com/MystenLabs/sui", branch = "devnet" }
 ```
+TODO: Give them a little more guidance on what this file is and where to find it. We describe `Move.toml` more fully at:
```

Comment:
this is intentionally public viewable?

Reply:
> this is intentionally public viewable?

Yep. Let's have Patrick review before merge.

Reply:
I am expecting the audience of this doc knows the basic of Rust and `Cargo.toml`, I don't think we need more descriptions here?

Reply:
> I am expecting the audience of this doc knows the basic of Rust and `Cargo.toml`, I don't think we need more descriptions here?

Great. I removed the TODO and linked into the Cargo doc.